types/storybook__addon-knobs/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for @storybook/addon-knobs 3.0
+// Project: https://github.com/storybooks/storybook
+// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="storybook__react" />
+
+// TODO: Once TS2.3 is released,
+// https://github.com/Microsoft/TypeScript/issues/14819 should be fixed.
+// Then, upgrade this package's typescript version to 2.3 and
```

Comment:
Tried a `rm -rf node_modules` and clean install to make sure its not an outdated `dtslint` version.

Full command:

```
$ npm run lint storybook__addon-knobs

> definitely-typed@0.0.1 lint /Users/joscha/Development/DefinitelyTyped
> dtslint types "storybook__addon-knobs"

Installing to /Users/joscha/Development/DefinitelyTyped/node_modules/dtslint/typescript-installs/2.3...
npm notice created a lockfile as package-lock.json. You should commit this file.

Installed!
/Users/joscha/Development/DefinitelyTyped/types/storybook__addon-knobs/index.d.ts
ERROR: 8:32  expect  TypeScript compile error: Cannot find module '@storybook/react'.

/Users/joscha/Development/DefinitelyTyped/types/storybook__addon-knobs/storybook__addon-knobs-tests.tsx
ERROR: 4:27  expect  TypeScript compile error: Cannot find module '@storybook/react'.
ERROR: 16:8  expect  TypeScript compile error: Cannot find module '@storybook/addon-knobs'.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! definitely-typed@0.0.1 lint: `dtslint types "storybook__addon-knobs"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the definitely-typed@0.0.1 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/joscha/.npm/_logs/2017-06-22T21_00_18_487Z-debug.log
```